lib/rbs/trace/inline_comment_visitor.rb

```diff
@@ -58,6 +58,17 @@ def find_method_definition(name)
           member.is_a?(AST::Members::MethodDefinition) && member.name == name
         end
       end
+
+      def with_context(node)
```

Comment:
CI is failing due to missing type declarations, so add them and run `rake rbs_inline`.

```suggestion
      # @rbs (Prism::ModuleNode | Prism::ClassNode) { () -> void } -> void
      def with_context(node)
```